lambda/business-wallet/batch-transactions/frontend/.env

```diff
@@ -0,0 +1,2 @@
+// App ID of your MetaKeep App from the developer console
+REACT_APP_APP_ID=9cc98bca-da35-4da8-8f10-655b3e51cb9e
```

Comment:
should this be empty. else we can use the app_id we use in the documentation.

Reply:
we can hardcode this to the prod app id once we deploy to prod